docs/quickstart.rst

```diff
@@ -63,4 +63,18 @@ Note that you don't have to pass ``request.form`` to Flask-WTF; it will
 load automatically. And the convenience ``validate_on_submit`` will check
 if it is a POST request and if it is valid.
 
+If your forms include validation, you'll need to add to your template to display
+any error messages.  Using the ``form.name`` field from the example above, that
+would look like this:
+
+.. sourcecode:: html+jinja
+
+    {% if form.name.errors %}
+        <ul class=errors>
```

Comment:
Use proper HTML instead.

```
<ul class="errors">
```

Reply:
I'm fine with either. Armin is adamant that we don't use quotes or closing tags if we don't have to in the Flask docs.

Reply:
Whoops, sorry for the typo.  I've amended the commit and force-pushed, so it should be sorted now.  Thanks